testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -4579,6 +4598,16 @@ public void testDeleteWithVarcharPredicate()
         }
     }
 
+    @Test
+    public void testDeleteTableNameInExplain()
```

Comment:
you can merge with testUpdateTableNameInExplain, testInsertTableNameInExplain, etc.

```
skipTestUnless(hasBehavior(CREATE_TABLE));

try (TestTable table = new TestTable(getQueryRunner()::execute, "test_delete_name_", "(col INTEGER)")) {
  if (hasBehavior(...)) {
    assertExplain(...);
  }
  if (hasBehavior(...)) {
    assertExplain(...);
  }
  if (hasBehavior(...)) {
    assertExplain(...);
  }
}
```